examples/Rudaina_work_area/Type-Based Directory/genzsprtwo_27_dir_orgnizer.py

```diff
@@ -0,0 +1,28 @@
+import os
+import shutil
+import glob
+
+DIRECTORY = "./dir"
+
+
+def dir_organizer(source_dir):
+    new_txt_path = os.path.join(source_dir, "./Text")
+    new_img_path = os.path.join(source_dir, "./Images")
+    miscellaneous = os.path.join(source_dir, "./Miscellaneous")
+    for directory in [new_img_path, new_txt_path, miscellaneous]:
+        if not os.path.exists(directory):
+            os.makedirs(directory)
+
+    files = glob.glob(os.path.join(source_dir, '*'))
+
+    for file in files:
```

Comment:
I believe this part is organizing the files so we can separate it out into another function. We can do this to achieve high cohesion and low coupling.
Or in simple terms "reduce dependency" and encourage single responsibility.